ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/trace/specs/trace-call/trace_call_14_1_vmTrace.json

```diff
@@ -94,19 +94,6 @@
           },
           "pc" : 31,
           "sub" : null
-        }, {
```

Comment:
This should not be happening.  We need to fing the code that is dropping this trace line when the error kind changes and fix it.

Reply:
Hi @shemnon, thanks for the feedback. As I mentioned in the PR description, I knew the line was not included in the trace due to the fact that any frame that halts with a reason different from INVALID_JUMP_DESTINATION or INSUFFICIENT_GAS is ignored. I've made the necessary adjustments now.